torch/_inductor/scheduler.py

```diff
@@ -2096,7 +2096,11 @@ def _init(self, nodes: list[ir.Operation]) -> None:
                 OrderedSet(V.graph.get_output_names()),
             )
         if config.reorder_for_compute_comm_overlap:
-            self.nodes = comms.reorder_compute_and_comm_for_overlap(self.nodes)
+            self.nodes = comms.reorder_compute_and_comm_for_overlap(
+                self.nodes,
+                OrderedSet(V.graph.graph_inputs.keys()),
+                OrderedSet(V.graph.get_output_names()),
+            )
```

Comment:
Is there a reason to not just call this inside of `reorder_compute_and_comm_for_overlap` and avoid the churn ?